Project trust - general settings (#10499)
qgis/QGIS-Documentation: docs/user_manual/introduction/qgis_configuration.rst

    embed auxiliary data`: the auxiliary data is stored in a separate :file:`.qgd`
    file along with the project file.

.. _project_trust:

**Project Trust**

> These options control how QGIS handles execution of embedded Python code contained

Reviewer: ```suggestion
These options control how QGIS handles execution of embedded scripts
```
It is [not only about Python](https://github.com/qgis/QGIS/pull/63693#discussion_r2467666004)

in project files. Embedded code includes macros, custom expression functions,
Python-based actions and attribute form initialization code. QGIS allows you to

> manage trust on a per-project or per-folder basis.

Reviewer: Do we need to insist that the trust is given to the whole project file and not per-script (allowing this and not that script in the project?

Reviewer: that was the intention.

.. _figure_project_trust_settings:

.. figure:: img/project_trust_general.png
   :align: center

   Project Trust Settings

* :guilabel:`Behavior for untrusted project's embedded Python code`:
  Use the dropdown menu to select how QGIS should respond when opening a project
  whose trust status has not yet been decided. Options include:

>   * :guilabel:`Never Execute`: block execution of all embedded Python code.

Reviewer: ```suggestion
  * :guilabel:`Never Execute`: block execution of all embedded scripts.
```

  * :guilabel:`Never Ask for Trust`: execute all embedded Python code automatically for trusted projects or folders,

>     without prompting. Denied projects are still blocked.

Reviewer: ```suggestion
    without prompting. Umdetermimed or already Denied projects are still blocked.
```
Or replace undetermined by new?

  * :guilabel:`Ask for Trust`: QGIS will display a trust dialog listing all embedded code and
    prompt you to allow or deny execution.
  * :guilabel:`Always Execute (Not recommended)`: enable execution of all embedded Python code.

* :guilabel:`Trusted projects and folders allowed Python code execution`:
  This list shows project files or folders that you have explicitly marked as
  trusted. Projects located in these paths are allowed to run their embedded

>   Python code without prompting. Use the |symbologyAdd|:sup:`Add new trusted project` or

Reviewer: ```suggestion
  scrits without prompting. Use the |symbologyAdd| :sup:`Add new trusted project` or
```

  |symbologyRemove|:sup:`Remove trusted project or folder` buttons to manage the list.

> * :guilabel:`Denied projects and folders Python code execution`:

Reviewer: ```suggestion
* :guilabel:`Untrusted projects and folders denied embedded script execution`:
```
Should this be denying?

  Paths listed here are explicitly marked as untrusted. Projects stored in these
  locations will never run embedded Python code. Use the |symbologyAdd|:sup:`Add new project denial` or
  |symbologyRemove|:sup:`Remove denied project or folder` buttons to
  maintain the list.

> When opening a project containing embedded Python code and whose trust state is

Reviewer: > but trust settings have not been configured

There is no such a situation. The "Behavior for embedded scripts within projects of undetermined trust" option is always set, and its default value is "Ask for trust", thus this behavior you get if you never touched that option. If you create a new user profile, turn the option from "Ask for trust" to something else and open for the first time a project with embedded scripts, you shouldn't get the same behavior. So this paragraph belongs to that option (and maybe add to that option that it is the default).
